SOOP/SOOP_BA/incoming_handler.sh

```diff
@@ -42,14 +42,13 @@ delete_previous_versions() {
 
     if has_extension $file "nc"; then
         del_function='s3_del'
-    elif has_extension $file "png"; then
+    elif has_extension $file "png" ||  has_extension $file "csv"; then
         local file_basename=`basename $file`
```

Comment:
`file_basename` is the same as `basename_file` above (line 35). Probably don't need both?